demos/demo_solo12.cpp

```diff
@@ -61,25 +65,36 @@ static THREAD_FUNCTION_RETURN_TYPE control_loop(void* args)
         sliders_filt_buffer[i].clear();
     }
 
-    robot.acquire_sensors();
-    map_sliders(robot.get_slider_positions(), sliders_zero);
+    robot->acquire_sensors();
+    map_sliders(robot->get_slider_positions(), sliders_zero);
 
     rt_printf("control loop started \n");
 
+    rt_printf("start calibration \n");
+
+    // Calibrate the robot.
+    blmc_robots::Vector12d joint_index_to_zero =
+        thread_data_ptr->joint_index_to_zero;
+
+    thread_data_ptr->robot->calibrate(joint_index_to_zero);
```

Comment:
should we pause here? to allow the user to understand that the calibration is going to be played?

Reply:
I rather like it without but can see that it can be confusing for the user. I will add it again.

Reply:
Done.

@MaximilienNaveau , can you double check and if things are okay now approve the PR?